pyomo/scripting/plugins/solve.py

```diff
@@ -49,23 +44,27 @@ def create_temporary_parser(solver=False, generate=False):
     parser = argparse.ArgumentParser(formatter_class=CustomHelpFormatter)
     _subparsers = parser.add_subparsers()
     _parser = _subparsers.add_parser('solve')
-    _parser.formatter_class=CustomHelpFormatter
+    _parser.formatter_class = CustomHelpFormatter
     if generate:
         #
         # Adding documentation about the two options that are
         # defined in the initial parser.
         #
-        _parser.add_argument('--generate-config-template',
+        _parser.add_argument(
+            '--generate-config-template',
             action='store',
             dest='template',
             default=None,
-            help='Create a configuration template file in YAML or JSON and exit.')
+            help='Create a configuration template file in YAML or JSON and exit.',
+        )
     if solver:
-        _parser.add_argument('--solver',
+        _parser.add_argument(
+            '--solver',
             action='store',
             dest='solver',
             default=None,
-            help='Solver name.  This option is required unless the solver name is specified in a configuration file.')
+            help='Solver name.  This option is required unless the solver name is specified in a configuration file.',
```

Comment:
I'm surprised it didn't line wrap this long string and the ones on L123 and L130